pages.it/linux/adduser.md

```diff
@@ -1,6 +1,7 @@
 # adduser
 
 > Servizio per aggiungere utenti.
+> Maggiori informazioni: <https://manpages.debian.org/latest/adduser/adduser.html>.
```

Comment:
Since you already get redirected, we could also delete the `.html` part to make the link even simpler. What do you think @bl-ue @navarroaxel? But I'd probably do that in a seperate PR, since there are many of those links.

Reply:
Hrm, I'd suggest @marchersimon that we should keep the URL as close to the final target URL as possible to avoid future breakages.